### DISCUSSION TITLE
Peak2Gene Link Calculation

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#621] [Questions / Documentation] [answered]
Hi again!
I have a question regarding this plot in the paper.

<img width="264" alt="Screen Shot 2021-03-11 at 4 55 02 PM" src="https://user-images.githubusercontent.com/19545198/110875500-97def080-828a-11eb-95df-a8aa40193eae.png">

In the second panel, may I ask how the correlation between TSS and E3 was calculated? The way I thought, was that the expression of the gene seems to be correlated with the changes in the accessibility of enhancer6, but when I look at the plot, to me it looks as if the peak in the TSS is correlated to the enhancer 6 peak. I would very much appreciate it if you could clarify this. Thank you

**Comment by rcorces** [comment, marked as answer]
The correlation for Peak2Gene links is between peak accessibility and gene expression. The figure is somewhat confusing because we showed the promoter peak in the diagram - but in my defense it would be equally as confusing to not show the promoter peak. The bottom dot plot panels I think illustrate it best with gene expression being correlated to the peak accessibility of E6. So your initial intuition was correct. Apologies for the confusion.
